components/kyma-environment-broker/common/orchestration/dto.go

```diff
@@ -104,15 +104,27 @@ type TargetSpec struct {
 	Exclude []RuntimeTarget `json:"exclude,omitempty"`
 }
 
+type KymaDetailResponse struct {
+	KymaVersion string `json:"kymaVersion,omitempty"`
+}
+
+type ClusterDetailResponse struct {
+	KubernetesVersion   string `json:"kubernetesVersion,omitempty"`
+	MachineImage        string `json:"machineImage,omitempty"`
+	MachineImageVersion string `json:"machineImageVersion,omitempty"`
+}
+
 type StatusResponse struct {
-	OrchestrationID string         `json:"orchestrationID"`
-	Type            Type           `json:"type"`
-	State           string         `json:"state"`
-	Description     string         `json:"description"`
-	CreatedAt       time.Time      `json:"createdAt"`
-	UpdatedAt       time.Time      `json:"updatedAt"`
-	Parameters      Parameters     `json:"parameters"`
-	OperationStats  map[string]int `json:"operationStats,omitempty"`
+	OrchestrationID string                 `json:"orchestrationID"`
+	Type            Type                   `json:"type"`
+	State           string                 `json:"state"`
+	Description     string                 `json:"description"`
+	CreatedAt       time.Time              `json:"createdAt"`
+	UpdatedAt       time.Time              `json:"updatedAt"`
+	Parameters      Parameters             `json:"parameters"`
+	OperationStats  map[string]int         `json:"operationStats,omitempty"`
+	KymaDetails     *KymaDetailResponse    `json:"kymaDetails,omitempty"`
+	ClusterDetails  *ClusterDetailResponse `json:"clusterDetails,omitempty"`
```

Comment:
is this sufficient? I would assume you need to populate those fields somewhere too
https://github.com/kyma-project/control-plane/blob/88243fac522464d9348c2e276fa1ddf614f7c1cb/components/kyma-environment-broker/internal/orchestration/handlers/converter.go#L13-L24